.github/workflows/deployDemo.yml

```diff
@@ -60,8 +52,6 @@ jobs:
       # which prevents the default bucket policy from applying to these objects
       - name: Upload to S3
         run: |
-          aws s3 sync ./demoRepo/workspaces/ui/build/ "s3://$BUCKET/" \
-            --delete \
+          aws s3 sync workspaces/ui/build/ "s3://$BUCKET/" \
             --sse=AES256 \
             --acl=public-read
-          aws cloudfront create-invalidation --distribution-id "$CLOUDFRONT_DIST" --paths "/*"
```

Comment:
getting `--delete` from the s3 sync and the cloudfront invalidations working needs a little IAM love than i expected it too. for now im removing them to unblock this.